browser/ui/views/tabs/vertical_tab_utils.h

```diff
@@ -13,6 +13,10 @@ class BrowserFrame;
 
 namespace tabs::utils {
 
+// This switch disables vertical tab strip regardless of the pref. This could be
+// useful when vertical tab strip causes browser to crash on start up.
+constexpr char kDisableVerticalTabsSwitch[] = "disable-vertical-tabs";
```

Comment:
nit: I think switch should be included `switches` namespace.
If there is no proper switches.h file for this, creating new one or using `brave/components/constants/brave_switches.h` would be fine.

Reply:
Yeah, I think `//brave/browser/ui/views/tabs/switches.h` is fine